### PR TITLE
[FIX] Ikonsynk Figma

### PR DESCRIPTION
**Pull request by KenAJoh**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

**Comment by andnorda** [comment]
Diskuteres her: https://nav-it.slack.com/archives/G7NE6ESLX/p1654172550795419

File: @navikt/core/icons/meta.json [hunk 1]
```diff
@@ -737,6 +737,12 @@
     "created_at": "2020-07-01T10:37:43.021Z",
     "pageName": "Status"
   },
+  {
+    "name": "ErrorColor",
```
Comment (andnorda) [inline @navikt/core/icons/meta.json]: Er dette navnet hakket bedre?
```suggestion
    "name": "ErrorColored",
```

Comment (KenAJoh) [reply @navikt/core/icons/meta.json]: Ja syntes det var bedre. Må fikses i Figma da evt før ny synk.

**Review by andnorda — APPROVED** [review]
LGTM

**Pull request merged**